packages/cli/templates/scaffolder-module.yaml

```diff
@@ -0,0 +1,14 @@
+description: A module exporting custom actions for @backstage/plugin-scaffolder-backend
+template: ./scaffolder-module
+targetPath: plugins
+backendModulePrefix: true
+prompts:
+  - id: id
+    prompt: Enter the ID of the plugin [required]
+    default: scaffolder
+    validate: backstage-id
+  - moduleid
```

Comment:
nit: I appreciate these are case-insensitive, but perhaps worth aiming to keep the casing right wherever we can to help with debugging issues stemming from places where the same string is used in a case-sensitive context (e.g. in the file immediately below).